en/flight_modes/position_mc.md

```diff
@@ -25,18 +25,24 @@ Landing in this mode is easy:
 1. The vehicle will lower propeller thrust, detect the ground and automatically disarm by default (see [COM_DISARM_LAND](../advanced_config/parameter_reference.md#COM_DISARM_LAND))
 
 :::warning
-In case the vehicle fails to stop horizontally: Keep it under control in [Altitude](../flight_modes/altitude_mc.md) mode and land like instructed above but manually making sure the vehicle stays above the landing spot using roll and pitch stick. After landing check GPS and magnetometer orientation, calibration.
-
-In case the vehicle fails to detect the ground and landing: After the vehicle is on the ground switch to [Manual/Stabilized](../flight_modes/manual_stabilized_mc.md) mode keeping the throttle stick low and manually disarm using a gesture or other command. Alternatively you can also use the kill switch when the vehicle is already on the ground.
+While very rare on a well calibrated vehicle, sometimes there may be problems with landing.
+- If the vehicle does not stop moving horizontally:
+  - You can still land under control in [Altitude mode](../flight_modes/altitude_mc.md).
+    The approach is the same as above, except that you manually ensure that the vehicle stays above the landing spot using the roll and pitch stick.
+  - After landing check GPS and magnetometer orientation, calibration.
+- If the vehicle does not detect the ground/landing and disarm:

```

Comment:
Probably it would be helpfull to link to https://docs.px4.io/master/en/advanced_config/prearm_arm_disarm.html#auto-disarming on the words "automatically disarm". Gives you the right pointers to continue reading how it works.

Reply:
Thanks. Agree. Fixed in another PR.